python/Ganga/testlib/GangaUnitTest.py

```diff
@@ -48,6 +48,7 @@ def start_ganga(gangadir_for_test, extra_opts=[]):
         ('Configuration', 'gangadir', gangadir_for_test),
         ('Configuration', 'user', 'testframework'),
         ('Configuration', 'repositorytype', 'LocalXML'),
+        ('Configuration', 'UsageMonitoringMSG', False),  # Turn off spyware
```

Comment:
We can indeed turn it on for the parallel tests if we want. It was not really a bottleneck at all but rather it just seemed silly that we were swamping the gangamon system with fake reports all the time.